forest/bonsai/tests/test_simulate_gps_data.py

```diff
@@ -271,3 +277,63 @@ def test_get_path_distance(coords1, coords2, directions1, mocker):
 def test_get_path_close_locations(coords1, coords3):
     """Tests case distance of locations is less than 250 meters."""
     assert len(get_path(coords1, coords3, "foot", "mock_api_key")[0]) == 2
+
+
+@pytest.fixture
+def random_path(coords1, coords2, directions1, mocker):
+    mocker.patch(
+        "openrouteservice.Client.directions", return_value=directions1
+    )
+
+    return get_path(coords1, coords2, "car", "mock_api_key")[0]
+
+
+def test_get_basic_path_simple_case(random_path):
+    """Test simple case of getting basic path"""
+    basic_random_path = get_basic_path(random_path, "car")
+    boolean_matrix = basic_random_path == np.array(
+        [
+            [51.458498, -2.59638],
+            [51.456975, -2.597508],
+            [51.460035, -2.60116],
+            [51.459923, -2.607755],
+            [51.457619, -2.608466],
+        ]
+    )
+    assert np.sum(boolean_matrix) == 10
+
+
+def test_get_basic_path_len_bicycle(random_path):
```

Comment:
It's good to be descriptive and explicit with test names, so `test_get_basic_path_length_by_bicycle`?